Directory.Build.props

```diff
@@ -2,5 +2,6 @@
 <Project>
   <PropertyGroup>
     <TargetNetFxVersion>net462</TargetNetFxVersion>
+    <TargetNetCoreVersion>netcoreapp3.1</TargetNetCoreVersion>
```

Comment:
This cannot be true for source build. We need to build with net6.0 or preferably net7.0

Reply:
Maybe naming is poor, it's actually "min supported version". In case of source build this isn't used.